src/core_modules/capture-core/components/FiltersForTypes/Date/DateFilter.component.js

```diff
@@ -257,12 +259,28 @@ class DateFilterPlain extends Component<Props, State> implements UpdatableFilter
         return !values || DateFilter.isFilterValid(values.main, values.from, values.to, values.start, values.end);
     }
 
-    getUpdatedValue(valuePart: { [key: string]: string }) {
-        // $FlowFixMe[cannot-spread-indexer] automated comment
+    // eslint-disable-next-line complexity
+    getUpdatedValue(valuePart: Object) {
         const valueObject = {
             ...this.props.value,
             ...valuePart,
         };
+        const dateFormat = systemSettingsStore.get().dateFormat;
+
+        if (valuePart.from && valueObject?.from?.value) {
+            valueObject.from = {
+                ...valueObject.from,
+                value: moment(convertLocalToIsoCalendar(valueObject.from.value)).format(dateFormat),
```

Comment:
In the working list, when the format is `DD-MM-YYYY`, an error is displayed when selecting a date from the calendar.
<img width="1211" alt="Screenshot 2024-12-10 at 12 37 10" src="https://github.com/user-attachments/assets/7148832b-337c-4efe-a687-a06614cfec1d">

Can you have a look?